app/javascript/packs/navigation/network_buttons_template.vue

```diff
@@ -16,6 +16,9 @@
 </template>
 
 <script>
+  import { mapActions } from 'vuex'
+  import types from '../mutations/main_mutations_types.js'
```

Comment:
perhaps it should be called actions instead of types?

Reply:
I was following by the documentation pattern - https://vuex.vuejs.org/guide/mutations.html#using-constants-for-mutation-types

Maybe we could leave it as is.